agent/credentials/manager.go

```diff
@@ -38,6 +38,10 @@ const (
 
 	v1CredentialsEndpointRelativeURIFormat = "%s?" + CredentialsIDQueryParameterName + "=%s"
 	v2CredentialsEndpointRelativeURIFormat = "%s/%s"
+
+	// The role types specified in the credentials payload msg from the backend
+	ApplicationRoleType = "TaskApplication"
+	ExecutionRoleType   = "TaskExecution"
```

Comment:
This needs a comment as well:
`"// ExecutionRoleType is ..."`

Reply:
fixed.